components/announcement_bar/configuration_bar/configuration_bar.jsx

```diff
@@ -119,9 +121,11 @@ class ConfigurationAnnouncementBar extends React.PureComponent {
         ) {
             const emailMessage = formatMessage({
                 id: AnnouncementBarMessages.PREVIEW_MODE,
-                defaultMessage: 'Preview Mode: Email notifications have not been configured',
+                defaultMessage: 'Preview Mode: Email notifications have not been configured. [AAE-000-0011](!{link})',
+                values={{
+                    link: AAE_REGISTRY_LINK,
+                }}
             });
-
```

Comment:
@jespino I tried adding the link here, but I am quite certain I'm using the `values` field incorrectly here, causing the build to break. Same below. Would you have an example of how I should approach updating this constant?